assets/stylesheets/style.scss

```diff
@@ -0,0 +1,946 @@
+  }
+}
+
+
+.diagram-detail > .content {
+  line-height: 1.375
+}
+
+
+//QUESTION SECTION
+
+.question {
+  padding-top: 75px;
+  padding-bottom: 140px;
+}
+
+
+//MEET FLO SECTION
+
+.meet-flo {
+  padding-top: 145px;
+}
+
+.flo-img {
+  background-image: url("../../public/img/florence_2.png");
+  width: 282px;
+  height: 305px;
+  top: -140px;
+  right: 0;
+  margin: 0 auto;
+  left: 0;
+}
+
+//ABOUT FLO SECTION
+
+.conversation-card {
+  background-color: white;
+  margin-bottom: 100px;
+
+  .narrow-padding {
+    z-index: 1;
+    position: relative;
+  }
+}
+
+.about-flo {
+  display: flex;
+  flex-direction: column;
+}
+
+.about-flo-icon {
+  width: 100px;
+  height: 200px;
+  top: -62px;
+  right: 0;
+  left: 0;
+  margin: 0 auto;
+}
+
+.about-flo-icon__bg {
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  top: 0;
+}
+
+.conversation-card.-cup {
+  .about-flo-icon {
+    background-image: url('../../public/img/section-1-illo.png');
+  }
+  .about-flo-icon__bg {
+    background-image: url('../../public/img/section-1-illo-background.jpg');
+    width: 312px;
+    height: 126px;
+    transform: rotate(180deg);
+  }
+}
+
+.conversation-card.-pills {
+  .about-flo-icon {
+    background-image: url('../../public/img/pills_3.png');
+    width: 80px;
+  }
+  .about-flo-icon__bg {
+    background-image: url('../../public/img/section-2-illo-background.jpg');
+    width: 330px;
+    height: 186px;
+    transform: rotate(180deg);
+  }
+}
+
+.conversation-card.-temp {
+  .about-flo-icon {
+    background-image: url('../../public/img/section-3-illo.png');
+    width: 80px;
+  }
+  .about-flo-icon__bg {
+    background-image: url('../../public/img/section-3-illo-background.jpg');
+    width: 402px;
+    height: 166px;
+  }
+}
+
+
+.conversation-card.-box {
+  .about-flo-icon{
+    background-image: url('../../public/img/section-4-illo.png');
+    width: 260px;
+      top: -40px;
+  }
+  .about-flo-icon__bg {
+    background-image: url('../../public/img/section-4-illo-background.jpg');
+    width: 558px;
+    height: 221px;
+  }
+}
+
+.conversation-card.-money {
+  margin-bottom: 140px;
+
+  .about-flo-icon {
+    background-image: url('../../public/img/section-5-illo.png');
+  }
+  .about-flo-icon__bg {
+      background-image: url('../../public/img/section-5-illo-background.jpg');
+      width: 536px;
+      height: 187px;
+    }
+}
+
+
+
+
+.conversation-card.-cheer {
+  margin-bottom: 0px;
+
+  .about-flo-icon {
+    background-image: url('../../public/img/section-6-illo.png');
+    width: 195px;
+    top: -100px;
+  }
+  .about-flo-icon__bg {
+    background-image: url('../../public/img/section-6-illo-background.jpg');
+    width: 396px;
+    height: 216px;
+    transform: rotate(180deg);
+  }
+}
+
+//BEHIND THE SCENES SECTION
+
+.flo-diagram {
+  position: relative;
+  background-image: url("../../public/img/giant-florence-of-doom.png");
+  width: 160px;
+  height: 300px;
+  margin: 30px auto 0;
+}
+
+
+//FOOTER
+
+footer > button {
```

Comment:
This button is better off with a class so you don't have to write such a specific selector or worry about side effects.